File: 151003/Mytnik/Lab2/Lab2/Lab2/DTO/TweetRequestTo.cs
using Lab2.DTO.Interface;
using Lab2.Models;

namespace Lab2.DTO
{
    public class TweetRequestTo(int Id, int? CreatorId, string? Title, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public int? CreatorId { get; set; } = CreatorId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;


    }
}

File: 151001/Pastukhou/RV/src/main/java/by/bsuir/kirillpastukhou/impl/dto/TweetRequestTo.java
package by.bsuir.kirillpastukhou.impl.dto;

import lombok.Getter;
import lombok.Setter;

import java.sql.Date;

@Setter
@Getter
public class TweetRequestTo {
    private long id;
    private long creatorId;
    private String title;
    private String content;
    private Date created;
    private Date modified;

}
